rosette/api.py

```diff
@@ -460,19 +460,24 @@ def call(self, parameters):
         @return: A python dictionary expressing the result of the invocation.
         """
 
-        if not isinstance(parameters, _DocumentParamSetBase):
-            if self.suburl != self.api.endpoints['NAME_SIMILARITY'] \
-                    and self.suburl != self.api.self.api.endpoints['NAME_TRANSLATION'] \
-                    and self.suburl != self.api.self.api.endpoints['NAME_DEDUPLICATION'] \
-                    and self.suburl != self.api.self.api.endpoints['ADDRESS_SIMILARITY']:
-                text = parameters
-                parameters = DocumentParameters()
-                parameters['content'] = text
-            else:
+        if type(parameters) == dict:
+            raise RosetteException(
+                "incompatible",
+                "Dictionary input is not supported",
```

Comment:
I think it would be helpful if the error message included that DocumentParameters (or string) should be used